src/m365/tenant/commands/people/people-profilecardproperty-add.ts

```diff
@@ -0,0 +1,170 @@
+        }
+
+        const excludeOptions: string[] = ['name', 'displayName', 'debug', 'verbose', 'output'];
+        const unknownOptions = Object.keys(args.options).filter(key => !excludeOptions.includes(key));
+
+        if (!propertyName.startsWith('customattribute') && unknownOptions.length > 0) {
+          return `Unknown options like ${unknownOptions.join(', ')} are only supported with customAttributes`;
+        }
+
+        if (propertyName.startsWith('customattribute')) {
+          const wronglyFormattedOptions = unknownOptions.filter(key => !key.toLowerCase().startsWith('displayname-'));
+          if (wronglyFormattedOptions.length > 0) {
+            return `Wrong option format detected for the following option(s): ${wronglyFormattedOptions.join(', ')}'. When adding localizations for customAttributes, use the format displayName-<languageTag>.`;
+          }
+        }
+
+        return true;
+      }
+    );
+  }
+
+  public allowUnknownOptions(): boolean | undefined {
+    return true;
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    const directoryPropertyName = profileCardPropertyNames.find(n => n.toLowerCase() === args.options.name.toLowerCase());
+
+    if (this.verbose) {
+      await logger.logToStderr(`Adding '${directoryPropertyName}' as a profile card property...`);
+    }
+
+    const requestOptions: any = {
+      url: `${this.resource}/v1.0/admin/people/profileCardProperties`,
+      headers: {
+        'content-type': 'application/json',
+        accept: 'application/json;odata.metadata=none'
+      },
+      responseType: 'json',
+      data: {
+        directoryPropertyName,
+        annotations: this.getAnnotations(args.options)
+      }
+    };
+
+    try {
+      const response: any = await request.post(requestOptions);
+
+      // Transform the output to make it more readable
+      if (args.options.output && args.options.output !== 'json' && response.annotations.length > 0) {
+        const annotation = response.annotations[0];
+
+        response.displayName = annotation.displayName;
+        annotation.localizations.forEach((l: { languageTag: string, displayName: string }) => {
+          response[`displayName ${l.languageTag}`] = l.displayName;
+        });
+
+        delete response.annotations;
+      }
+
+      await logger.log(response);
+    }
+    catch (err: any) {
+      this.handleRejectedODataJsonPromise(err);
+    }
+  }
+
+  private getAnnotations(options: Options): { displayName: string, localizations?: { languageTag: string, displayName: string }[] }[] {
+    if (!options.displayName) {
+      return [];
+    }
+
+    return [
+      {
+        displayName: options.displayName!,
+        localizations: this.getLocalizations(options)
+      }
+    ];
+  }
+
+  private getLocalizations(options: Options): { languageTag: string, displayName: string }[] {
+    const excludeOptions: string[] = ['name', 'displayName', 'debug', 'verbose', 'output'];
```

Comment:
this list is used twice. Here and in validation. How about we move it outside of being local method variable and change it to global for this command